mlflow/pyfunc/__init__.py

```diff
@@ -963,7 +963,7 @@ def spark_udf(spark, model_uri, result_type="double", env_manager="local"):
         # Prepare restored environment in driver side if possible.
         if env_manager == "conda":
             _get_flavor_backend(local_model_path, no_conda=False, install_mlflow=False).prepare_env(
-                model_uri=local_model_path
+                model_uri=local_model_path, stream_output=False
```

Comment:
Are we relying on the fact that passing `stream_output=False` to `mlflow.utils.process.exec_cmd()` captures and returns the stdout / stderr streams? I don't see those streams being used anywhere.

The naming is a bit confusing. `stream_output=False` would seem to imply that the command output shouldn't be sent anywhere at all.

Reply:
`capture_output` might make more sense.